src/rest-server/src/models/v2/job/k8s.js

```diff
@@ -557,12 +552,6 @@ const generateTaskRole = (
                 path: `/var/log/pai`,
               },
             },
-            {
-              name: 'job-ssh-secret-volume',
-              secret: {
-                secretName: 'job-ssh-secret',
-              },
-            },
```

Comment:
How dose the job container get the ssh secret after this change? We don't mount the secret as a file?

Reply:
It will in `/usr/local/pai/user-extension-secrets/userExtensionSecrets.yaml`, please see fix 53 of runtime which related to this.

https://github.com/microsoft/openpai-runtime/pull/53/files#diff-e21e1d55082d72593bd006834666b4a67bdff33b61d86a9346d2b4f1160141e7R56-R75